code/modules/reagents/reagents/other.dm

```diff
@@ -505,3 +505,23 @@
 /datum/reagent/other/rejuvenating_agent/touch_obj(obj/O)
 	if(istype(O))
 		O.make_young()
+
+/datum/reagent/resuscitator
+	name = "Resuscitator"
+	id = "resuscitator"
+	description = "Incredibly rare cardiac stimulant."
+	reagent_state = LIQUID
+	color = "#A0522D"
+	metabolism = 0.5
+	overdose = REAGENTS_OVERDOSE
+	scannable = 1
```

Comment:
Should this chem that can revive the dead really be dupeable?

Reply:
I was thinking it's making it visible.